app/AppKernel.php

```diff
@@ -56,15 +56,14 @@ public function registerBundles()
             // PrestaShop Translation parser
             new PrestaShop\TranslationToolsBundle\TranslationToolsBundle(),
             // REST API consumer
-            new Csa\Bundle\GuzzleBundle\CsaGuzzleBundle(),
+            new EightPoints\Bundle\GuzzleBundle\EightPointsGuzzleBundle(),
```

Comment:
I think this is an important thing to mention 😄

Reply:
Mentionned! 😉